corehq/apps/export/migrations/0011_auto_20210112_1727.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.16 on 2021-01-12 17:27
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
```

Comment:
i would rename the migration file name a little more meaningful vs the autogenerated name but other than that looking good